src/util/sortBy.test.js

```diff
@@ -0,0 +1,138 @@
+// @flow
+
+import sortBy from "./sortBy";
+
+describe("util/sortBy", () => {
+  const numericSample = [4, 2, 1, 8, 16];
```

Comment:
nit: I would deepFreeze these examples so they can't be modified by test code.

Reply:
done